README.md

```diff
@@ -7,1334 +7,81 @@
-meant to be used when connecting to a docker container.
-
-```sh
-$ fauna endpoint add
-? Endpoint name localhost
-? Database URL http://localhost:8443
-? Database Secret secret
-Checking secret... done
-? Make this endpoint default [no]
-Saved endpoint localhost to ~/.fauna-shell
-```
-
-The endpoint name is an arbitrary name that can be used in the `--endpoint` flag
-of other commands. The database URL should typically be `http://localhost:8443`
-or `https://db.fauna.com`, although it can be set to any URL to a fauna
-instance. The database secret is the secret used to authenticate with that
-database.
-
-An endpoint can be set as the default, in which case `fauna shell` and
-`fauna eval` will choose that endpoint by default.
-
-Endpoints can be listed with the `endpoint list` command like this:
-
-```sh
-$ fauna endpoint list
-  localhost
-* cloud
-```
-
-There we see that the `cloud` endpoint has a `*` next to its name, meaning that
-it's the current default one.
-
-Finally, endpoints will be saved to a `~/.fauna-shell` file like this:
-
-```ini
-default=cloud
-
-[endpoint.localhost]
-url=http://localhost:8443
-secret=secret
-
-[endpoint.cloud]
-secret=FAUNA_SECRET_KEY
-```
-
-# Connecting to local endpoints
-
-If you are running Fauna locally using our Docker image, you will need to
-configure the CLI to work with local endpoints so you can interact with the
-database running in the Docker container.
-
-The docker container is explained in depth here:
-https://docs.fauna.com/fauna/current/tools/dev.
-
-Once you've installed the Shell and logged in, you can configure it by doing the
-following:
-
-By default, the Fauna Docker image serves data via port 8443. To add a
-connection to this port, run `fauna endpoint add`. The Database URL should be
-`http://localhost:8443` (the default), and the database secret should be
-`secret` by default.
-
-```sh
-$ fauna endpoint add
-? Endpoint name localhost
-? Database URL http://localhost:8443
-? Database Secret secret
-Checking secret... done
-? Make this endpoint default [no]
-Saved endpoint localhost to ~/.fauna-shell
-```
-
-Now, you can interact with your local database through the Fauna Shell by
-running the command below:
-
-```sh
-fauna shell --endpoint localhost
+npm update -g fauna-shell
 ```
 
-# Overriding Connection Parameters
-
-Most commands support the following options. You can specify them if you want to
-connect to a local instance of Fauna.
+## Documentation
 
-```sh
-OPTIONS
-  --endpoint=name      Selects an endpoint from ~/.fauna-shell
-  --endpointURL=domain Overrides the `url` setting in the selected endpoint.
-  --secret=secret      Overrides the `secret` setting in the selected endpoint.
-  --timeout=timeout    [default: 5000] Connection timeout in milliseconds
-```
+For usage instructions and a full [list of
+commands](https://docs.fauna.com/fauna/current/build/cli/commands/), see the
+[Fauna docs](https://docs.fauna.com/fauna/current/build/cli/).
```

Comment:
we should talk about what process/tools we could have to keep these in sync moving forwards

Reply:
My current process is to have a GitHub watch set up for this repo so that I'm aware of any PRs that come in.

I also get plugged in to planned working during sprint planning.

Happy to discuss other possibilities, but that's my MO for other repos (core, drivers, etc.). CC @ecooper.